components/channel_members_dropdown/channel_members_dropdown.jsx

```diff
@@ -123,7 +123,7 @@ export default class ChannelMembersDropdown extends React.Component {
         }
         const canMakeUserChannelMember = canChangeMemberRoles && isChannelAdmin;
         const canMakeUserChannelAdmin = canChangeMemberRoles && isLicensed && isMember;
-        const canRemoveUserFromChannel = canRemoveMember && !channel.group_constrained && (!isDefaultChannel || isGuest);
+        const canRemoveUserFromChannel = canRemoveMember && !(channel.group_constrained && !user.is_bot) && (!isDefaultChannel || isGuest);
```

Comment:
Not sure if you agree with me on this but do you think it makes sense to change this to
```suggestion
        const canRemoveUserFromChannel = canRemoveMember && (!channel.group_constrained || user.is_bot) && (!isDefaultChannel || isGuest);
```
I feel like it would be to get what the constraints are more quickly rather than wanting to evaluate within the parenthesis and then doing inverse on it. This way I know I can remove a member if it's not group constrained or if it is group constrained, the user needs to be a bot. Thoughts?